addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -641,6 +641,7 @@ export class Rtc extends Record {
 
     async toggleMicrophone() {
         if (this.localSession.isMute) {
+            await this.undeafen();
             await this.unmute();
         } else {
             await this.mute();
```

Comment:
`unmute` after `undeafen` without conditional is inefficient and probably can lead to bugs: I don't see any early return and it posts message to all tabs so that the tab making the call intercepts and make the action.

Should probably ensure a single action is made, and it picks the right action. `undeafen()` necessarily implies `unmute()`, we functionally don't allow undeafen without unmute because this is a useless transition.
_(I have sometimes the bug to manually require unmute after undeafen, but that's definitely a bug)_